internal/server/iigointernal/legislature.go

```diff
@@ -266,3 +268,27 @@ func (l *legislature) incurServiceCharge(cost shared.Resources) bool {
 	}
 	return ok
 }
+
+func sameIslandIDSlice(x, y []shared.ClientID) bool {
```

Comment:
I think you can use `reflect.DeepEqual`

Reply:
reflect.DeepEqual considers the order, I think. I'm doing this in case islands give back a different order

Reply:
I have a sorter function for `ClientID`

Reply:
`internal\common\shared\shared.go`, `SortByClientID`

Reply:
`sort.Sort(shared.SortByClientID(lst))`

Reply:
(note also that slices are passed by ref so you might want to copy before sorting :))